demos/camera_demo/src/colortracking.cpp

```diff
@@ -91,19 +93,21 @@ int ColorTracking::process(FrameBuffer data, size_t length) {
         // object not found
         return 0;
     }
-    float angularPosition = getAngularPosition();
+
+    float angularPosition = mCallbacks.mGet();
```

Comment:
Small change. Can you move the declaration of angularPosition inside `if(mImageSlidingWindow)`. The angular position only needed for the sliding window.